vault/resource_kmip_secret_backend.go

```diff
@@ -0,0 +1,207 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-provider-vault/util"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+var kmipAPIFields = []string{
+	"listen_addrs", "server_hostnames", "server_ips", "tls_ca_key_type", "tls_ca_key_bits",
```

Comment:
nit: might be a bit easier to read if each field was on its own line (sorted).

Reply:
Fixed in 124fd7f